src/journal-remote/journal-remote-main.c

```diff
@@ -1136,6 +1145,13 @@ static int run(int argc, char **argv) {
                 s.check_trust = !arg_trust_all;
         }
 
+        memset(&s.metrics, 0xFF, sizeof(s.metrics));
```

Comment:
This is not necessary. As `journal_reset_metrics()` does the similar.

Reply:
done